src/EnergyPlus/ChillerElectricEIR.cc

```diff
@@ -2183,11 +2183,10 @@ namespace ChillerElectricEIR {
             }
 
             if (this->CondenserType == EvapCooled) {
-                Real64 RhoAir = Psychrometrics::PsyRhoAirFnPbTdbW(
-                    DataEnvironment::StdBaroPress, condInletTemp, DataLoopNode::Node(this->CondInletNodeNum).HumRat, RoutineName);
+                Real64 const RhoWater = Psychrometrics::RhoH2O(DataGlobals::InitConvTemp);
```

Comment:
Replaced moist air density with water density (at 5.05 deg C). Air density should not have been used.